src/main/java/racingcar/global/util/RandomNumberGenerator.java

```diff
@@ -0,0 +1,20 @@
+package racingcar.global.util;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class RandomNumberGenerator {
+    public static final int MIN_NUMBER = 0;
+    private static final int MAX_NUMBER = 9;
+    private static RandomNumberGenerator instance;
+
+    public static RandomNumberGenerator getInstance() {
+        if (instance == null) {
+            instance = new RandomNumberGenerator();
+        }
+        return instance;
+    }
+
+    public int generator() {
+        return Randoms.pickNumberInRange(MIN_NUMBER, MAX_NUMBER);
+    }
```

Comment:
클래스 이름이  _RandomNumberGenerator_ 이니까 `randomGenerator.generator()` 보단 `randomGenerator. generateNumber()` 정도가 더 명확하고 자연스러운 것 같아요 !

```suggestion
    public int generateNumber() {
        return Randoms.pickNumberInRange(MIN_NUMBER, MAX_NUMBER);
    }
```